hibernate-envers/src/main/java/org/hibernate/envers/configuration/internal/metadata/BasicMetadataGenerator.java

```diff
@@ -6,20 +6,26 @@
  */
 package org.hibernate.envers.configuration.internal.metadata;
 
+import java.util.Iterator;
 import java.util.Properties;
 
+import org.dom4j.Element;
```

Comment:
Please do not re-order imports unnecessarily.  This creates unnecessary diffs in the files.  In general, `org.dom4j.` imports are listed below `org.hibernate` and in a separate section on purpose.  This helps call attention to classes that use this library as it is something we plan to remove in the next major release.